interchaintest/misbehaviour_test.go

```diff
@@ -0,0 +1,320 @@
+package interchaintest_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+	"time"
+
+	simappparams "cosmossdk.io/simapp/params"
+	"github.com/cometbft/cometbft/crypto/tmhash"
+	cometproto "github.com/cometbft/cometbft/proto/tendermint/types"
+	cometprotoversion "github.com/cometbft/cometbft/proto/tendermint/version"
+	comettypes "github.com/cometbft/cometbft/types"
+	cometversion "github.com/cometbft/cometbft/version"
+	sdked25519 "github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
+	"github.com/cosmos/cosmos-sdk/std"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	transfertypes "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
+	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	ibctypes "github.com/cosmos/ibc-go/v7/modules/core/types"
+	ibccomettypes "github.com/cosmos/ibc-go/v7/modules/light-clients/07-tendermint"
+	ibctesting "github.com/cosmos/ibc-go/v7/testing"
+	ibcmocks "github.com/cosmos/ibc-go/v7/testing/mock"
+	"github.com/cosmos/ibc-go/v7/testing/simapp"
+	relayertest "github.com/cosmos/relayer/v2/interchaintest"
+	"github.com/dvsekhvalnov/jose2go/base64url"
+	"github.com/strangelove-ventures/interchaintest/v7"
+	"github.com/strangelove-ventures/interchaintest/v7/chain/cosmos"
+	"github.com/strangelove-ventures/interchaintest/v7/ibc"
+	"github.com/strangelove-ventures/interchaintest/v7/testreporter"
+	"github.com/strangelove-ventures/interchaintest/v7/testutil"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+func TestScenarioMisbehaviourDetection(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+
+	t.Parallel()
+
+	numVals := 1
+	cf := interchaintest.NewBuiltinChainFactory(zaptest.NewLogger(t), []*interchaintest.ChainSpec{
```

Comment:
we could add `numFullNodes := 0` to remove the full nodes to further simplify resources for this test case.